### ISSUE TITLE
Running `pod install` throws error "`RNShareMenu` does not specify a Swift version"

### ISSUE THREAD

**Issue by i6mi6** [Expensify/react-native-share-menu#54]
After installing the package and then doing `cd ios && pod install` I get the following error:

```
[!] Unable to determine Swift version for the following pods:

- `RNShareMenu` does not specify a Swift version and none of the targets (`MyProject`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.
```

Adding `SWIFT_VERSION` to my project passes pods installation however it throws tons of swift related errors on build. 

Any idea how to properly install this package?

**Comment by Gustash** [comment]
Sorry about that, this will be fixed on the next version

**Comment by caiosba** [comment]
Fixed in v4.0.1
